packages/clay-shared/src/useFocusManagement.ts

```diff
@@ -43,7 +43,7 @@ export function isFocusable({
 
 	if (
 		(tabIndex != null && tabIndex >= minTabIndex) ||
-		!!contentEditable === true
+		contentEditable === true
```

Comment:
I think contentEditable can also be a string (not saying the coercion makes sense with a string), so we might want to check for both the boolean and string value of true?
```suggestion
		contentEditable === 'true' || contentEditable === true
```

Reply:
Oh yeah, this is probably because we also get it now directly from the DOM.

Reply:
> I think contentEditable can also be a string

Ha, good catch. I need to revise my operator precendence table. 😂